docs/index.rst

```diff
@@ -173,6 +173,7 @@ ar71xx-generic
 * TP-Link
 
   - Archer C5 (v1) [#ath10k]_
+  - Archer C59 (v1) [#80211s]_
   - Archer C7 (v2) [#ath10k]_
```

Comment:
ar71xx-generic devices that should have the ath10k remark

```sh
% grep "ATH10K_PACKAGES" * -B
[...]
ar71xx-generic-alias openmesh-mr1750v2
ar71xx-generic:packages $ATH10K_PACKAGES
--
ar71xx-generic-alias openmesh-om5p-acv2
ar71xx-generic:packages $ATH10K_PACKAGES
--
ar71xx-generic-device tp-link-archer-c5-v1 archer-c5-v1
ar71xx-generic:packages $ATH10K_PACKAGES
--
ar71xx-generic-device tp-link-archer-c7-v2 archer-c7-v2
ar71xx-generic:packages $ATH10K_PACKAGES
--
ar71xx-generic-device tp-link-archer-c7-v4 archer-c7-v4
ar71xx-generic:packages $ATH10K_PACKAGES
--
ar71xx-generic-device tp-link-archer-c25-v1 archer-c25-v1 # BROKEN: OOM with 5GHz enabled in most environments
ar71xx-generic:packages $ATH10K_PACKAGES_QCA9887
--
ar71xx-generic-device tp-link-re450 re450
ar71xx-generic:packages $ATH10K_PACKAGES
--
ar71xx-generic-alias ubiquiti-unifi-ac-lr
ar71xx-generic:packages $ATH10K_PACKAGES
--
ar71xx-generic-device ubiquiti-unifi-ac-pro ubnt-unifiac-pro
ar71xx-generic:packages $ATH10K_PACKAGES
[...]
```

Reply:
of those only the RE450 missed it, or did i miss something?